system-test/pubsub.ts

```diff
@@ -103,11 +103,11 @@ describe('pubsub', () => {
     const subscriptionStream = pubsub
       .getSubscriptionsStream()
       .on('data', deleteTestResource);
-    const snapshotStream = pubsub
-      .getSnapshotsStream()
-      .on('data', deleteTestResource);
+    // const snapshotStream = pubsub
```

Comment:
Why commented out? Should the lines be removed?

Reply:
Whoops, I needed to add a note here. List snapshots doesn't work with the emulator so I commented it out as a temporary workaround.